src/app/@core/models/Group.model.ts

```diff
@@ -0,0 +1,8 @@
+export class Group<T> {
```

Comment:
rename this file follow naming convention

Reply:
changed file name to: group